newIDE/app/src/AssetStore/NewObjectFromScratch.js

```diff
@@ -144,6 +144,15 @@ const getMergedInstalledWithDefaultEnumeratedObjectMetadataByCategory = ({
       {
         name: 'Scene3D::Model3DObject',
       },
+      {
+        name: 'TileMap::SimpleTileMap',
+        fullName: i18n._(t`Tile map`),
+        description: i18n._(t`Displays a tiled-based map.`),
+        iconFilename: 'JsPlatform/Extensions/tile_map.svg',
+        isRenderedIn3D: false,
+        assetStorePackTag: 'starter tilemaps',
```

Comment:
as this is a built-in object, you don't need all the props but just `name` and `assetStorePackTag` if I'm not mistaken?
The rest is taken from the object declaration in the JSExtension so it would be a doublon